numba/tests/test_linalg.py

```diff
@@ -1327,7 +1327,30 @@ def check(a, b, **kwargs):
             # check that the computed results are contig and in the same way
             self.assert_contig_sanity(got, "F")
 
-            np.testing.assert_array_almost_equal_nulp(got, expected, nulp=20)
+            use_reconstruction = False
+            # try plain match of the result first
+            try:
+                np.testing.assert_array_almost_equal_nulp(
+                    got, expected, nulp=10)
+            except Exception:
```

Comment:
`except AssertionError` should ensure we don't silence any errors in the test itself.